src/export/export-type-styles.ts

```diff
@@ -0,0 +1,69 @@
+import tokensImport from "../tokens.json";
+import { TokensJSON } from "./types";
+
+const exportTypeStyles = async () => {
+  // Set up the JSON object structure
+  const tokensJSON: TokensJSON = {
+    ...tokensImport,
+    "type-tokens": tokensImport["type-tokens"],
+  };
+
+  // Get all local type styles
+  const typeStyles = await figma.getLocalTextStylesAsync().then(
+    (typeStyles) => typeStyles,
+    (error) => {
+      console.error("Error fetching local text styles:", error);
+      return [];
+    }
+  );
+
+  // Create an empty object to hold the type styles
+  const typeStylesJSON: Record<string, {
```

Comment:
It looks like the type declarations for this object are set differently than in the file above. I would pick one style and stick with it for both files for consistency.